src/components/transactions/TxList/index.tsx

```diff
@@ -1,29 +1,41 @@
 import GroupedTxListItems from '@/components/transactions/GroupedTxListItems'
-import { groupConflictingTxs } from '@/utils/tx-list'
+import { groupTxs } from '@/utils/tx-list'
 import { Box } from '@mui/material'
-import type { TransactionListPage } from '@safe-global/safe-gateway-typescript-sdk'
+import type { Transaction, TransactionListPage } from '@safe-global/safe-gateway-typescript-sdk'
 import type { ReactElement, ReactNode } from 'react'
 import { useMemo } from 'react'
 import TxListItem from '../TxListItem'
 import css from './styles.module.css'
+import { uniq } from 'lodash'
```

Comment:
I'm not sure if our build is able to tree-shake lodash properly, it's better to help it:

```suggestion
import uniq from 'lodash/uniq'
```